lib/util/src/hse_err.c

```diff
@@ -118,27 +109,42 @@ merr_strerror(merr_t err, char *buf, size_t buf_sz)
 char *
 merr_strinfo(merr_t err, char *buf, size_t buf_sz, size_t *need_sz)
 {
-    ssize_t     sz = 0;
+    ssize_t     sz = 0, ret = 0;
     const char *file = NULL;
 
     if (err) {
         file = merr_file(err);
+
         if (file)
-            sz = snprintf(buf, buf_sz, "%s:%d: ", file, merr_lineno(err));
-        if (sz < 0) {
+            ret = snprintf(buf, buf_sz, "%s:%d: ", file, merr_lineno(err));
+
+        if (ret < 0) {
             sz = strlcpy(buf, "<failed to format error message>", buf_sz);
             goto out;
         }
+
+        sz += ret;
         if (sz >= buf_sz)
             goto out;
 
         sz += merr_strerror(err, buf + sz, buf_sz - sz);
+        if (sz >= buf_sz)
+            goto out;
+
+        ret = snprintf(buf + sz, buf_sz - sz, " (%d)", merr_errno(err));
```

Comment:
Good change!

Reply:
Would be easier to parse and less and cleaner code if we put the errno right after the file:line, use the format "%s:%d:%d" at line 119.

Reply:
A lot of tooling supports file:lineno these days quite well. Not sure how it would be handled given another :%d.

Reply:
No convinced either way...have not seen many "%s:%d:%d" out there. Would like to remain as is, as was the original proposal.